setup-env

```diff
@@ -111,6 +128,25 @@ else
 fi
 set -o nounset
 
+# List Python versions and select one interactively
+if [ $LIST_VERSIONS -ne 0 ]; then
+  echo Available Python versions:
+  pyenv versions --bare --skip-aliases --skip-envs
+  read -p -r "Enter the desired Python version: " PYTHON_VERSION
+fi
+
+# Check if PYTHON_VERSION isn't empty. If it is installed, set it locally.
+pyenv versions --bare --skip-aliases --skip-envs > $TMPFILE
+if [ -n "$PYTHON_VERSION" ]; then
+  if grep --fixed-strings --quiet "$PYTHON_VERSION" $TMPFILE; then
+    echo Using Python version "$PYTHON_VERSION"
+    pyenv local "$PYTHON_VERSION"
+  else
+    echo Error: Python version "$PYTHON_VERSION" is not installed.
```

Comment:
I don't _think_ the quotes around `$PYTHON_VERSION` are necessary in these lines.

Reply:
Linting threw up some issues with it.

![image](https://github.com/cisagov/skeleton-generic/assets/74698586/bf985dc3-246d-48b4-8bc0-74c06b2e319e)